v2/src/modules/address/components/TableRowNftTransfers.vue

```diff
@@ -147,7 +147,7 @@ const tokenImg = computed<string>(() => {
 })
 
 const totalTokens = computed<FormattedNumber | string>(() => {
-    return formatNumber(new BN(props.transfer.token).toNumber())
+    return formatNumber(new BN(props.transfer.token || 1).toNumber())
```

Comment:
Do we want to use 0 or 1 in this case? If the `props.transfer.token` returns 0, the totalTokens would still return 1